swagger.yaml

```
@@ -3540,6 +3539,271 @@ paths:
+            type: string
+            enum: [asc, desc]
+            default: asc
+          description: |
+            The ordering of items from the point of view of the blockchain,
+            not the page listing itself. By default, we return oldest first, newest last.
+      responses:
+        "200":
+          description: Return the tickers provided by the metadata oracle
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/nutlink_address_tickers"
+        "400":
+          $ref: "#/components/responses/bad_request"
+        "403":
+          $ref: "#/components/responses/unauthorized_error"
+        "404":
+          $ref: "#/components/responses/not_found"
+        "429":
+          $ref: "#/components/responses/overusage_limit"
+        "418":
+          $ref: "#/components/responses/autobanned"
+        "500":
+          $ref: "#/components/responses/internal_server_error"
+
+  /nutlink/{address}/{ticker}:
+    get:
+      tags:
+        - Nut.link
+      description: List of records of a specific ticker
+      parameters:
+        - in: path
+          required: true
+          name: address
+          schema:
+            type: string
+            description: Address of a metadata oracle
+        - in: path
+          required: true
+          name: ticker
+          schema:
+            type: string
+            description: Ticker for the pool record
+        - in: query
+          name: count
+          required: false
+          schema:
+            type: integer
+            maximum: 100
+            default: 100
+          description: The number of results displayed on one page.
+        - in: query
+          name: page
+          required: false
+          schema:
+            type: integer
+            default: 1
+          description: The page number for listing the results.
+        - in: query
+          name: order
+          required: false
+          schema:
+            type: string
+            enum: [asc, desc]
+            default: asc
+          description: |
+            The ordering of items from the point of view of the blockchain,
+            not the page listing itself. By default, we return oldest first, newest last.
+      responses:
+        "200":
+          description: Return the tickers provided by the metadata oracle
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/nutlink_address_ticker"
+        "400":
+          $ref: "#/components/responses/bad_request"
+        "403":
+          $ref: "#/components/responses/unauthorized_error"
+        "404":
+          $ref: "#/components/responses/not_found"
+        "429":
+          $ref: "#/components/responses/overusage_limit"
+        "418":
+          $ref: "#/components/responses/autobanned"
+        "500":
+          $ref: "#/components/responses/internal_server_error"
+
+  /nutlink/{ticker}/addresses:
```

```suggestion
  /nutlink/tickers/{ticker}/addresses:
```

Might also add additional `tickers` as the outher route collides with `/nutlink/{address}`